dev-build.cmd

```diff
@@ -0,0 +1,2 @@
+call grunt build
```

Comment:
Why do you need this patch at all?

You test it locally with IIS? If so, you should change the first line to `cmd /c` instead of call.